app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] It would be nice if you fix your associations here as well because the likes table has a foreign key `user_id` not the `author_id` so it would be nice if you make changes here accordingly :+1:

- [ ] It would be great if you fix your migration files as well because in the migrations you have author_id in the **comments** and **likes** tables :fearful: Kindly rename them to the `user_id` and kindly don't make any change with the post because post has the author_id so don't make any change with the post model or migration :+1: